src/index.js

```diff
@@ -1,5 +1,68 @@
 import 'bootstrap';
 import './scss/index.scss';
-import displayScoreArr from './js/displayLB.js';
+import tableData from './js/displayLB.js';
 
-displayScoreArr();
\ No newline at end of file
+const scoreForm = document.querySelector('#score-form');
+
+const getData = () => {
+  fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/VwaMYCtnuP7TunvNhm2w/scores/',
+    {
+      method: 'GET',
+    },
+  )
+    .then((res) => res.json())
+    .then((final) => {
+      const data = final.result;
+      data.forEach((val) => {
+        tableData(val.user, val.score);
+      });
+    });
+};
```

Comment:
- [ ] Kindly use `async` and `await` with the fetch method to send and receive API data as requested by the previous reviewer 👍 You can look at this [lesson](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/) for more information on that 💯

Reply:
yes, it is, Sorry I don't have much time for Know, but I'm working on it on the weekend.
Even if the previous commit was optional, I know I should change it soon.

Reply:
- [ ] Kindly use `async` and `await` with the fetch method to send and receive API data as requested by the previous reviewer. You can look at this [lesson](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/) for more information on that 💯. `This is an important requirement and you cant proceed to the next project without implementing this. Make sure to fix it.`